### ISSUE TITLE
Suggestion: Be able to change the root directory

### ISSUE THREAD

**Issue by Tom60chat** [0xJacky/nginx-ui#68]
Hello,
To be able to use nginx-ui behind a reverse proxy, it would be cool to be able to change the location / to whatever we want, like /nginx-ui/

For example:
https://myserver:9000/ will be https://myserver/nginx-ui/

**Comment by 0xJacky** [comment]
This feature will be released with `v1.7.3`.

Just modify your nginx configuation like the example below, and it will be automatically supported sub path deployment.
```
location /<sub-path>/ {
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection $connection_upgrade;
        proxy_pass http://127.0.0.1:9000/;
    }
```

**Comment by Tom60chat** [comment]
Wow, cool!

Thanks for your quick answer.
